feature-libs/cart/base/components/add-to-cart/add-to-cart.component.ts

```diff
@@ -147,18 +159,27 @@ export class AddToCartComponent implements OnInit, OnDestroy {
     this.quantity = 1;
 
     this.addToCartForm.controls['quantity'].setValue(1);
-
-    this.hasStock = Boolean(product.stock?.stockLevelStatus !== 'outOfStock');
-
     this.inventoryThreshold = product.stock?.isValueRounded ?? false;
-
-    if (this.hasStock && product.stock?.stockLevel) {
-      this.maxQuantity = product.stock.stockLevel;
-    }
-
     if (this.productListItemContext) {
       this.showQuantity = false;
     }
+
+    if (this.featureToggles.showRealTimeStockInPDP && this.sapUnit) {
```

Comment:
[MINOR]
This `if` condition contains both a feature toggle and a check of `sapUnit`. In a year from now, when we'll be removing this feature toggle from the code, the developer might be confused whether they should leave the `else` block or remove it. IMHO they should leave it - in case when `sapUnit` is false.

So let's leave a code comment for future self (or future other developer):
> When removing the feature toggle in the future, let's leave the `if-else` block.
> In case of absent `sapUnit` we want to fallback to the `stock` info from the `product` object.

Reply:
Done